### PR TITLE
Add guide of deploy tfjs-node on cloud

### PR DESCRIPTION
**Pull request by kangyizhang**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-website/321)

### CODE REVIEW

File: docs/guide/node_in_cloud.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Deploy tfjs-node project on cloud platform
+
+This doc describes how to run a Node.js process with @tensorflow/tfjs-node package on cloud platforms.
+
+Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require extra configurations. This guide will use the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in @tensorflow/tfjs-examples repository, which trains MNIST with tfjs-node package on Heroku and GCloud. The details of Heroku’s Node.js Support can be found in [Heroku Node.js Support](https://devcenter.heroku.com/articles/nodejs-support) article. The details of Node.js on Google Cloud Platform can be found on the [Google Cloud page](https://cloud.google.com/nodejs/docs/).
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require additional configuration. This guide will show how to run the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in the @tensorflow/tfjs-examples repository on Heroku and GCloud. Heroku’s Node.js support is documented in this [article](https://devcenter.heroku.com/articles/nodejs-support). Running Node.js on Google Cloud Platform is documented [here](https://cloud.google.com/nodejs/docs/).

File: docs/guide/node_in_cloud.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+3. Heroku CLI
+
+### Create the Node.js app
+
+1. Create a folder and copy the `data.js`, `main.js`, `model.js` and `package.json` files from the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example.
+2. Make sure the @tensorflow/tfjs-node dependency is @1.2.4 or newer version.
+
+### Build your app and run it locally
+
+1. Run the `npm install` command in your local directory to install the dependencies that are declared in the `package.json` file. You should be able to see the tfjs-node package is installed and libtensorflow is downloaded.
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: "... able to see that the tfjs-node package ...".

File: docs/guide/node_in_cloud.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+========================>----------------------------------------------------------------------------------: 35.5
+```
+
+3. Make sure you ignore build artifacts, such as node_modules, in your .gitignore file.
+
+### Create and deploy the heroku app
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: Heroku (with capital H).

File: docs/guide/node_in_cloud.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+1. Create a new app on heroku website
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: "Create a new app on the Heroku website".

File: docs/guide/node_in_cloud.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+remote:        CPU-linux-1.2.5.tar.gz
+remote:        * Downloading libtensorflow
+remote:
+remote:        * Building TensorFlow Node.js bindings
+remote:        added 92 packages from 91 contributors and audited 171 packages in 9.983s
+remote:        found 0 vulnerabilities
+remote:
+```
+
+In the process logs on heroku you should be able to see the model training logs:
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: "In the process logs on Heroku, you should be able to see the model training logs."

File: docs/guide/node_in_cloud.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+Total params: 594922
+Trainable params: 594922
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+====>--------------------------------------------------------------------: 221.9
+```
+
+You can also start or debug the process in heroku [console](https://devcenter.heroku.com/articles/heroku-dashboard#application-overview).
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: Heroku (with capital H).

File: docs/guide/node_in_cloud.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+### Using tfjs-node prior to version 1.2.4
+
+If you are using tfjs-node package before version 1.2.4, the package requires g++ to compile the node native addon from source files. You will have to make sure your stack has Linux build-essential package (newer version stack may not have it on default).
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: "your stack has the Linux build-essential package" (add "the" before Linux).

File: docs/guide/node_in_cloud.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+## Deploy Node.js project on Google Cloud Platform
+
+###Prerequisite:
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: ### Prerequisites
(plural, remove colon)

File: docs/guide/node_in_cloud.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+1. Have a valid Google Cloud Project with billing account
+2. Install Google Cloud [client tool](https://cloud.google.com/storage/docs/gsutil_install)
+3. Add app.yaml file to configure the [Node.js Runtime](https://cloud.google.com/appengine/docs/flexible/nodejs/runtime)
+
+### Deploy app to GCloud
+
+Run `gcloud app deploy` to deploy the local code and configurations to App Engine. In the deploy logs you should be able to see tfjs-node is installed:
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: ... "able to see that tfjs-node is installed" ... (add that).

File: docs/guide/node_in_cloud.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+Step #1:
+Step #1: CPU-linux-1.2.5.tar.gz
+Step #1: * Downloading libtensorflow
+Step #1:
+Step #1: * Building TensorFlow Node.js bindings
+Step #1: added 88 packages from 85 contributors and audited 171 packages in 13.392s
+Step #1: found 0 vulnerabilities
+```
+
+And in the apps logs, you should be able to see the model training process:
```
Comment (brijk7) [inline docs/guide/node_in_cloud.md]: In the app logs,... (remove "And").

Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 9 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 8 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done

File: docs/guide/node_in_cloud.md (hunk 7 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 6 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 5 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 4 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 3 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 2 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

File: docs/guide/node_in_cloud.md (hunk 1 above)
Comment (kangyizhang) [reply docs/guide/node_in_cloud.md]: Done.

**Pull request merged**